### PR TITLE
Update 2023-02-21-23.0.0.2-beta.adoc

### PR DESCRIPTION
**Pull request by mbroz2**
@k8vance88 @aknguyen7 can you confirm that no other changes need to be made to the examples in order to be able to bump the features up to their latest versions?

### CODE REVIEW

**Review by aknguyen7 — COMMENTED** [review]
I did not have appSecurity-4.0 (nor appSecurity-2.0) in my env and it still works. Tested with appSecurity-4.0 & mpOpenApi-3.0 works as well.

File: posts/2023-02-21-23.0.0.2-beta.adoc [hunk 1]
```diff
@@ -70,11 +70,13 @@ To enable database connection testing, the following minimum set of features mus
...
 ----
     <feature>adminCenter-1.0</feature>
-    <feature>appSecurity-2.0</feature>
+    <feature>appSecurity-4.0</feature>
     <feature>restConnector-2.0</feature>
-    <feature>mpOpenApi-1.0</feature>
+    <feature>mpOpenApi-3.0</feature>
 ----
 
+Although this example uses the `mpOpenApi-3.0` and `appSecurity-4.0` features, you can use any version of the MicroProfile OpenAPI and Application Security (Jakarta Security) features.
```
Comment (mbroz2) [inline posts/2023-02-21-23.0.0.2-beta.adoc]:
```suggestion
Although this example uses the `mpOpenApi-3.0` and `appSecurity-4.0` features, you can use any valid version combination of the MicroProfile OpenAPI and Application Security (Jakarta Security) features.

```

**Review by aknguyen7 — CHANGES_REQUESTED** [review]
@mbroz2 i think we should remove the appSecurity feature (if it's still okay to make change), i looked at the docs here https://github.com/OpenLiberty/docs/issues/6228 and it's corrected, we don't have it included, I updated the GA Blog to not include that as well. Thanks

File: posts/2023-02-21-23.0.0.2-beta.adoc [hunk 2]
```diff
@@ -70,11 +70,14 @@ To enable database connection testing, the following minimum set of features mus
 [source, xml]
 ----
     <feature>adminCenter-1.0</feature>
-    <feature>appSecurity-2.0</feature>
+    <feature>appSecurity-4.0</feature>
```
Comment (mbroz2) [inline posts/2023-02-21-23.0.0.2-beta.adoc]:
```suggestion
```

File: posts/2023-02-21-23.0.0.2-beta.adoc [hunk 3]
```diff
@@ -87,10 +90,10 @@ The following sample `server.xml` file enables the features to configure the Adm
     <!-- Enable features -->
     <featureManager>
         <feature>adminCenter-1.0</feature>
-        <feature>appSecurity-2.0</feature>
+        <feature>appSecurity-4.0</feature>
```
Comment (mbroz2) [inline posts/2023-02-21-23.0.0.2-beta.adoc]:
```suggestion
```

File: posts/2023-02-21-23.0.0.2-beta.adoc [hunk 4]
```diff
@@ -70,11 +70,14 @@ To enable database connection testing, the following minimum set of features mus
...
 ----
     <feature>adminCenter-1.0</feature>
-    <feature>appSecurity-2.0</feature>
+    <feature>appSecurity-4.0</feature>
     <feature>restConnector-2.0</feature>
-    <feature>mpOpenApi-1.0</feature>
+    <feature>mpOpenApi-3.0</feature>
 ----
 
+Although this example uses the `mpOpenApi-3.0` and `appSecurity-4.0` features, you can use any valid version combination of the MicroProfile OpenAPI and Application Security (Jakarta Security) features.
```
Comment (mbroz2) [inline posts/2023-02-21-23.0.0.2-beta.adoc]:
```suggestion
Although this example uses the `mpOpenApi-3.0`, you can use any MicroProfile OpenAPI version that is compatible with your other features..

```

File: posts/2023-02-21-23.0.0.2-beta.adoc [hunk 5]
```diff
@@ -70,11 +70,14 @@ To enable database connection testing, the following minimum set of features mus
...
-    <feature>appSecurity-2.0</feature>
     <feature>restConnector-2.0</feature>
-    <feature>mpOpenApi-1.0</feature>
+    <feature>mpOpenApi-3.0</feature>
 ----
 
+Although this example uses the `mpOpenApi-3.0`, you can use any MicroProfile OpenAPI version that is compatible with your other features..
+
+
+
```
Comment (mbroz2) [inline posts/2023-02-21-23.0.0.2-beta.adoc:79]:
```suggestion
```

File: posts/2023-02-21-23.0.0.2-beta.adoc [hunk 6]
```diff
@@ -70,11 +70,13 @@ To enable database connection testing, the following minimum set of features mus
 [source, xml]
 ----
     <feature>adminCenter-1.0</feature>
-    <feature>appSecurity-2.0</feature>
     <feature>restConnector-2.0</feature>
-    <feature>mpOpenApi-1.0</feature>
+    <feature>mpOpenApi-3.0</feature>
 ----
 
+Although this example uses the `mpOpenApi-3.0`, you can use any MicroProfile OpenAPI version that is compatible with your other features..
```
Comment (mbroz2) [inline posts/2023-02-21-23.0.0.2-beta.adoc]:
```suggestion
Although this example uses the `mpOpenApi-3.0` feature, you can use any MicroProfile OpenAPI version that is compatible with your other features.
```

**Review by dmuelle — APPROVED** [review]
LGTM

**Pull request merged**